mount-partitions.sh

```diff
@@ -19,38 +19,55 @@ export DBUS_SYSTEM_BUS_ADDRESS="${DBUS_SYSTEM_BUS_ADDRESS:-unix:path="${ROOT_MOU
 # Partition is only the label, e.g. boot, state, data
 dbus_get_mount() {
     part="$1"
-
     result=$(dbus-send --system --print-reply \
         --dest=org.freedesktop.systemd1 /org/freedesktop/systemd1/unit/mnt_2d${part}_2emount org.freedesktop.DBus.Properties.Get \
         string:"org.freedesktop.systemd1.Mount" string:"What" | grep "string" | cut -d'"' -f2 2>&1)
-    # If the output doesn't match the /dev/* device regex, exit with an error
+    # If the output doesn't match the /dev/* device regex, return empty and do not exit
     if [ "$(echo "${result}" | grep -E '^/dev/')" = "" ]; then
-        echo "ERROR: Could not determine ${part} device from dbus. Please launch Supervisor as a privileged container with DBus socket access."
-        exit 1
+        echo ""
+    else
+        echo "${result}"
     fi
-
-    echo "${result}"
 }
 
 # Get the current boot block device in case there are duplicate partition labels
 # for `(balena|resin)-(boot|state|data)` found.
+secure_boot_partitions='efi
+rpi
+'
```

Comment:
I find this hard to read, is there any reason to have these on separate lines rather than just `secure_boot_partitions='efi rpi'`?